### ISSUE TITLE
Strange SSR issue

### ISSUE THREAD

**Issue by xiao-hu** [theKashey/react-imported-component#51]
Hi,

  I followed the instructions for SSR (not streaming) in README. I only replaced `ReactDOM.render` with `ReactDOM.hydrate`, e.g.,:

```
  import { rehydrateMarks } from 'react-imported-component';

  // this will trigger all marked imports, and await for competition.
  rehydrateMarks().then(() => {
    ReactDOM.hydrate(<App />,document.getElementById('main'));
  });
```

I saw a react warning `react-dom.development.js:180 Warning: Did not expect server HTML to contain a <div> in <div>.` indicating some difference between the client/server side rendering. 

So I went ahead and investigate what the difference:
```
  import { rehydrateMarks } from 'react-imported-component';

  // this will trigger all marked imports, and await for competition.
  rehydrateMarks().then(() => {
    const divMain = document.getElementById('main')
    console.log(divMain.innerHTML); #server side result looks OK
    ReactDOM.hydrate(<App />, divMain);
    console.log(divMain.innerHTML); #async components are not rendered
  });
```
It turns out that the async components are not rendered. I passed in empty `options` to `importedComponent `. The server side generated the marks:
`<script>window.___REACT_DEFERRED_COMPONENT_MARKS=["1v1uipo"];/*stream 0*/</script>`. If I pass `LoadingComponent` in the `options`, then `LoadingComponent` will be rendered.

Other than this, everything else worked just fine.

Checked the process against README a few times and still couldn't figure out why :(.
Do I need to set`options.render` or use `whenComponentsReady`?

**Comment by xiao-hu** [comment]
It turns out that the issue is caused by the extra loadables created by step 2 to 4 (`import importedComponents from 'src/imported';`).
These extra loadables load the same chunks as these created by `importedComponent` do. But they have separate loading status. `rehydrateMarks` loads the chunks using one loadable and the other loadable with outdated loading status still thought the chunks are not loaded. Hence the issue.

Removing step 2 to step 4 fixes the issue. Everything works well so far. Not sure if it has side effects that I haven't noticed yet.

**Comment by theKashey** [comment]
Does #52 solved your issue?

**Comment by theKashey** [comment]
> These extra loadables load the same chunks as these created by importedComponent do. 
Fixed in 5.3.0. It was tracking only one _loadable_ per one module.

**Comment by xiao-hu** [comment]
Thanks!. Verified that the issues are fixed. Closing...

> Does #52 solved your issue?

It solves the `printDrainHydrateMarks returns empty array` issue.

> Something is wrong with a loading tracking. Strange, it was checked so many times.

Looks like it's time to improve the tests to better cover it :) .
